src/v2/Apps/FairOrginizer/fairOrganizerRoutes.tsx

```diff
@@ -34,10 +36,18 @@ export const fairOrganizerRoutes: AppRouteConfig[] = [
     render: ({ Component, props }) => {
       if (Component && props) {
         const { fairOrganizer } = props as any
-        const { profile } = fairOrganizer
+        const { profile, runningFairs } = fairOrganizer
+
         if (!profile) {
           return <ErrorPage code={404} />
         }
+
+        const activeFair = extractNodes(runningFairs)[0]
+        if (activeFair) {
+          const { href } = activeFair as any
```

Comment:
**question**: I'm surprised to see `as any` here. Is `activeFair` not already typed?

Reply:
I haven't found any examples of typing props in routes so followed existing practice `props as any`. So `extractNodes(runningFairs)` _(which is of type `any` too)_ was returning `object[]`
But if we want to get props values typed we can extend props type with query response type